acme_diags/driver/streamflow_driver.py

```diff
@@ -123,7 +137,17 @@ def run_diag(parameter):
             if parameter.print_statements:
                 print('area_upstream dimensions={}'.format(area_upstream.shape))
         else:
-            data_mat = scipy.io.loadmat(parameter.test_mat_file)
+            files_in_test_data_path = os.listdir(parameter.test_data_path)
+            mat_files = list(filter(lambda file_name: file_name.endswith('.mat'), files_in_test_data_path))
```

Comment:
Note: if using `mat` files for `test`, then find all the `mat` files in this directory. There should only be one. We will use that one.